rpm-x64/Dockerfile

```diff
@@ -37,6 +41,13 @@ RUN yum -y install \
   bzip2-devel e2fsprogs-devel file-devel libacl-devel libattr-devel \
   libxml2-devel lzo-devel nss nss-devel popt-devel sharutils xz-devel
 
+# We install our own ruby, let's remove the system one. It made rvm fail to build ruby for some reason
+RUN yum remove -y ruby
+
+# Update curl with a statically linked binary
+COPY --from=CURL_GETTER /curl-amd64 /usr/bin/curl
```

Comment:
We may need to remove `curl-devel` from the list above

Reply:
We need `curl-devel` because we build `git` which needs it. It probably means `yum` is also installing the `curl` binary as a dependency of `curl-devel`, though.